### DISCUSSION TITLE
How about register projects to Hacktoberfest?

### DISCUSSION THREAD

**Discussion by 4513ECHO** [aquaproj/aqua#2293] [Deprecated (Proposal)] [answered]
[Hacktoberfest](https://hacktoberfest.com/), a supporting event for open source, will start from October 1st. How about register projects of aqua to it to increase the contributers?

**Comment by suzuki-shunsuke** [comment]
Thank you for your suggestion.
I'm not familiar with this event, so I'm looking over the website now.

**Reply by suzuki-shunsuke** [reply, marked as answer]
We looked over the website and understood the rules of Hacktoberfest.
We'll join Hacktoberfest.
We added the topic `hacktoberfest` to some repositories.

https://github.com/search?q=topic%3Ahacktoberfest+org%3Aaquaproj&type=Repositories

Even if some repositories won't have the topic `hacktoberfest`, please don't hesitate to contribute to them.
We'll add the topic `hacktoberfest` to repositories if you'll send pull requests to them.
If we'll forget to add the topic, please ask us to add it.

TODO:

- [x] Add `hacktoberfest` to our repositories
- [ ] Create or update CONTRIBUTING.md
- [ ] Write document to encourage your contribution
- [ ] Create issues for Hacktoberfest
- [ ] Add labels `hacktoberfest`, `good first issue`, or `help wanted` to some issues
- [ ] Advertize in X (Twitter)

**Reply by suzuki-shunsuke** [reply]
BTW, I also maintained other OSS such as [tfcmt](https://github.com/suzuki-shunsuke/tfcmt) and [tfaction](https://github.com/suzuki-shunsuke/tfaction), and the contribution to them is also welcome. 😄

**Reply by 4513ECHO** [reply]
Thanks! I will send PRs a lot!
